package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grafana-singlestat-panel",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
**Question:** as supported Grafana version has changed and is not supporting `7.x` anymore, should do a major version bump for the plugin as well (to `2.0.0`), due to this being a "breaking change"?

Reply:
Good point @leventebalogh, switched to v2.0.0